resources/js/components/headers/AlbumsHeader.vue

```diff
@@ -118,8 +119,17 @@ const emits = defineEmits<{
 	help: [];
 }>();
 
+//  'UPLOAD_PHOTO' => 'Upload Photo',
```

Comment:
Do you need the commentary here?

Reply:
![image](https://github.com/user-attachments/assets/bb36f7d7-4260-4ca2-aeae-ad50e4fcc729)

Reply:
Fixed [7ad3073](https://github.com/LycheeOrg/Lychee/pull/3049/commits/7ad307389239a77f87044ccd32a8efff70e2dfb6)